project/src/components/cardFilm/cardFilm.tsx

```diff
@@ -0,0 +1,130 @@
+type Id = number;
+type Title = string;
+type Img = string;
+
+type CardsFilmsList = {
```

Comment:
Здесь лучше типом сделать карточку, а ниже использовать как массив
```suggestion
type CardsFilm = {
```